epoxy-adapter/src/main/java/com/airbnb/epoxy/EpoxyViewHolder.java

```diff
@@ -20,21 +21,30 @@ public EpoxyViewHolder(ViewGroup parent, @LayoutRes int layoutId) {
   public void bind(@SuppressWarnings("rawtypes") EpoxyModel model, List<Object> payloads) {
     this.payloads = payloads;
 
+    if (epoxyHolder == null && model instanceof EpoxyModelWithHolder) {
+      epoxyHolder = ((EpoxyModelWithHolder) model).createNewHolder();
+      epoxyHolder.bindView(itemView);
+    }
+
     if (payloads.isEmpty()) {
       // noinspection unchecked
-      model.bind(itemView);
+      model.bind(objectToBind());
```

Comment:
You could do it with an if instanceof check then call the correctly typed bind method with the view holder or view right?